cmd/earthly/subcmd/cloud_installation_cmds.go

```diff
@@ -231,18 +242,78 @@ func (c *CloudInstallation) printTable(installations []cloud.Installation) {
 		if i.IsDefault {
 			selected = "*"
 		}
-		var description string
+		var coloredStatus string
 		switch i.Status {
-		case cloud.CloudStatusActive:
-			description = "Ready to use"
-		case cloud.CloudStatusConnected:
-			description = "Reachable, but not yet validated"
-		case cloud.CloudStatusProblem:
-			description = "Please contact Earthly for support"
+		case cloud.CloudStatusGreen:
+			coloredStatus = color.GreenString(i.Status)
+		case cloud.CloudStatusYellow:
+			coloredStatus = color.YellowString(i.Status)
+		case cloud.CloudStatusRed:
+			coloredStatus = color.RedString(i.Status)
+		default:
+			coloredStatus = color.HiRedString(i.Status)
+		}
+		suffix := ""
+		if i.StatusMessage != "" {
+			suffix = fmt.Sprintf(":%s", i.StatusMessage)
```

Comment:
Should there be a space here? 
```suggestion
			suffix = fmt.Sprintf(": %s", i.StatusMessage)
```